src/components/Navbar/MainNavbar.vue

```diff
@@ -8,7 +8,7 @@
       :condition="areBoardOptionsShown"
       element="board"
     />
-    <nav aria-label="main navigation" class="main-nav">
+    <nav aria-label="main navigation" class="main-nav w-full">
```

Comment:
Not sure if I really need this `w-full` class. I mean, it's a block-level element, it'll occupy the whole available width without it, won't it?